docs/attributes-registry/db.md

```diff
@@ -221,6 +220,7 @@
 | `db.connection_string` | string | Deprecated, use `server.address`, `server.port` attributes instead. | `Server=(localdb)\v11.0;Integrated Security=true;` | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>"Replaced by `server.address` and `server.port`." |
 | `db.elasticsearch.node.name` | string | Deprecated, use `db.instance.id` instead. | `instance-0000000001` | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Replaced by `db.instance.id`. |
 | `db.jdbc.driver_classname` | string | Removed, no replacement at this time. | `org.postgresql.Driver`; `com.microsoft.sqlserver.jdbc.SQLServerDriver` | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>Removed as not used. |
+| `db.user` | string | Deprecated, no replacement at this time. | `readonly_user`; `reporting_user` | ![Deprecated](https://img.shields.io/badge/-deprecated-red)<br>No replacement at this time. |
 <!-- endsemconv -->
 
 ### Deprecated Elasticsearch Attributes
```

Comment:
I'm wondering why title is specific to the elasticsearch? Probably a mistake?

Reply:
Now checking it twice, I see we have the same set of deprecated attributes with different naming - generic db and elasticsearch. Is it correct?

Reply:
See https://github.com/open-telemetry/semantic-conventions/pull/866#discussion_r1548086063